File: VB/CustomLabelsAndStatusesExample/Model/PaymentState.vb
#Region "#customstatususings"
Imports DevExpress.Mvvm.POCO
Imports System.Windows.Media
#End Region ' #customstatususings

Namespace CustomLabelsAndStatusesExample
    #Region "#customstatus"
    Public Class PaymentState
        Public Shared Function Create() As PaymentState
            Return ViewModelSource.Create(Function() New PaymentState())
        End Function

        Protected Sub New()
        End Sub
        Public Overridable Property Id() As Integer
        Public Overridable Property Caption() As String
        Public Overridable Property Brush() As Brush
    End Class
    #End Region ' #customstatus
End Namespace

File: CS/CustomLabelsAndStatusesExample/Model/PaymentState.cs
#region #customstatususings
using DevExpress.Mvvm.POCO;
using System.Windows.Media;
#endregion #customstatususings

namespace CustomLabelsAndStatusesExample {
    #region #customstatus
    public class PaymentState {
        public static PaymentState Create() {
            return ViewModelSource.Create(() => new PaymentState());
        }

        protected PaymentState() { }
        public virtual int Id { get; set; }
        public virtual string Caption { get; set; }
        public virtual Brush Brush { get; set; }
    }
    #endregion #customstatus
}
